src/config.js

```diff
@@ -205,7 +218,22 @@ const setConfig = async cfg => {
 
     config.logIndent = cfg.LOG_INDENT ? Number(cfg.LOG_INDENT) : 2;
 
+    config.metrics = {
+        disabled: cfg.METRICS_DISABLED.toLowerCase() === 'true',
```

Comment:
I assume these are being defaulted as per the section above if they are not set by the ENV?

Reply:
Yes it is